doc/contribute/bin_blobs.rst

```diff
@@ -224,5 +224,30 @@ the integration of such blob for the foreseeable future.
 Submission and review process
 =============================
 
-This section will be updated once the standard format for describing binary
-blobs is agreed upon.
+For references to binary blobs to be included in the project, they must be
+reviewed and accepted by the Technical Steering Committee (TSC).
```

Comment:
I get why we are doing this, but we may need to revisit at some point if the number of TSC discussions becomes onerous for frequently updated blobs.

Reply:
This process is only for new blobs, not for version updates to existing blobs! That's what I had in mind at least

Reply:
Oh, OK, that's not super clear to me from reading this document. Do you think you could add a sentence or modify an existing one to make that explicit?

Reply:
Yep, done.